### PR TITLE
Fix for 0.12

### PR DESCRIPTION
**Pull request by eric-corumdigital**

### CODE REVIEW

File: src/Data/Incremental/Record.purs [hunk 1]
```diff
@@ -2,18 +2,22 @@ module Data.Incremental.Record
...
 import Prelude
 
 import Data.Incremental (class Patch, Change, Jet, fromChange, patch, toChange)
-import Data.Monoid (class Monoid, mempty)
 import Data.Newtype (class Newtype, unwrap, wrap)
-import Data.Record as Record
 import Data.Symbol (class IsSymbol, SProxy(..))
-import Type.Row (class RowLacks, class RowToList, Cons, Nil, RLProxy(..), kind RowList)
+import Type.Row (class RowToList, Cons, Nil, RLProxy(..), kind RowList)
+import Record as Record
```
Comment (paf31) [inline src/Data/Incremental/Record.purs:19]: Unused?

**Comment by paf31** [comment]
Thanks! Looks like the Travis config needs updating too, but I can do that.

**Comment by eric-corumdigital** [comment]
I might have messed up the tuples version number… may need to be 5 rather than 4.1. I get confused as to whether bower/pulp is heeding bower.json or using a different (already installed) version.

File: src/Data/Incremental/Record.purs (hunk 1 above)
Comment (eric-corumdigital) [reply src/Data/Incremental/Record.purs:19]: Is what unused?

Comment (paf31) [reply src/Data/Incremental/Record.purs:19]: I didn't see this import being used.

Comment (paf31) [reply src/Data/Incremental/Record.purs:19]: Sorry, I see it now, ignore me.

**Review by paf31 — APPROVED** [review]

**Comment by paf31** [comment]
Thanks again!

**Pull request merged**